tests/test_model_generator.py

```diff
@@ -3395,7 +3395,7 @@ def test_modelgen_reflection_1(self):
             "tuple<a:std::str, b:tuple<c:std::int64, d:std::str>>",
         )
 
-    @tb.typecheck
+    # @tb.typecheck
```

Comment:
We should probably have a `typecheck_xfail` decorator

Reply:
ok